lua/mini/jump.lua

```diff
@@ -271,7 +275,7 @@ function MiniJump.expr_jump(backward, till)
   end
   H.update_state(target, backward, till, vim.v.count1)
 
-  return vim.api.nvim_replace_termcodes('v:<C-u>lua MiniJump.jump()<CR>', true, true, true)
+  return vim.api.nvim_replace_termcodes('v<cmd>lua MiniJump.jump()<CR>', true, true, true)
```

Comment:
Please, use `<Cmd>` instead of `<cmd>`. I try to consistently use it across all modules, as it is how it is written in Vim help docs.